.appveyor.yml

```diff
@@ -0,0 +1,48 @@
+# See https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/
+skip_branch_with_pr: true
+
+# AppVeyor will run on the output branch commits unless the first line of the
+# commit message contains '[ci skip]'
+# Enable 'Do not build on "Push" events' in the AppVeyor project settings
+# to only build commits from pull requests
+branches:
+  only:
+    - master
+
+only_commits:
+  files:
+    - .appveyor.yml
+    - content/
+
+image: ubuntu
+services:
+  - docker
+
+install:
+  - source ci/install.sh
+  - env
+    
+test_script:
+  - bash build/build.sh
+  - MANUSCRIPT_FILENAME=manuscript-$APPVEYOR_BUILD_VERSION-${APPVEYOR_REPO_COMMIT:0:7}.pdf
+  - cp output/manuscript.pdf $MANUSCRIPT_FILENAME
+  - appveyor PushArtifact $MANUSCRIPT_FILENAME
+
+build: off
+
+cache:
+  - ci/cache
+
+on_success:
+  - echo "Artifacts available from https://ci.appveyor.com/project/$APPVEYOR_REPO_NAME/builds/$APPVEYOR_BUILD_ID/artifacts"
```

Comment:
Can replace `https://ci.appveyor.com` with `$APPVEYOR_URL`.

`$APPVEYOR_REPO_NAME` should be `$APPVEYOR_ACCOUNT_NAME/$APPVEYOR_PROJECT_NAME` because here we need the AppVeyor account info rather than the GitHub account info.

Reply:
Done in bcd55532168cbe77ab4e0c9453539ae8b6f88aea